### PR TITLE
Enable screenshots of consoles and improve wait_serial on s390x

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
local verifications for both commits:
http://opeth.suse.de/tests/3526  and 29 following

File: backend/console_proxy.pm [hunk 1]
```diff
@@ -61,6 +61,9 @@ sub AUTOLOAD {
 	if (exists $wrapped_retval->{exception}) {
 	    die $wrapped_retval->{exception};
 	}
+    
+    # get more screenshots from consoles, especially from x3270 on s390 
+    $autotest::current_test->take_screenshot;
```
Comment (coolo) [inline backend/console_proxy.pm]: if this passes, @okurz managed to get rid of the tidy check on travis

Comment (coolo) [inline backend/console_proxy.pm]: I see - susanne disabled tidy for the file inline

**Comment by coolo** [comment]
merged and added 471a853 to it

**Pull request closed without merging**